crates/holochain/tests/tests/gossip/mod.rs

```diff
@@ -0,0 +1,327 @@
+#![allow(clippy::field_reassign_with_default)]
+#![allow(clippy::type_complexity)]
+#![allow(clippy::single_match)]
+
+use hdk::prelude::*;
+use holo_hash::ActionHash;
+use holochain::sweettest::SweetConductorBatch;
+use holochain::sweettest::SweetInlineZomes;
+use holochain::sweettest::{
+    await_consistency, DynSweetRendezvous, SweetAgents, SweetConductor, SweetConductorConfig,
+    SweetDnaFile, SweetLocalRendezvous,
+};
+use holochain::test_utils::inline_zomes::{simple_create_read_zome, simple_crud_zome};
+use holochain_conductor_api::conductor::ConductorConfig;
+use holochain_conductor_api::conductor::NetworkConfig;
+use holochain_zome_types::{
+    record::{Record, RecordEntry},
+    Entry,
+};
+
+#[cfg(feature = "test_utils")]
+#[tokio::test(flavor = "multi_thread")]
+#[cfg_attr(target_os = "macos", ignore = "flaky")]
+async fn bad_network_gossip() {
+    holochain_trace::test_run();
+
+    let vous = SweetLocalRendezvous::new_raw().await;
+
+    vous.drop_sig().await;
```

Comment:
We can't do that without dropping the bootstrap any more because it's all one task. This will mean the `true` passed below to enable bootstrapping won't do anything

Reply:
Yes, good reminder. I feel it's not worth keeping this test. It just demonstrates that connections don't work without a signal server.